.travis.yml

```diff
@@ -8,6 +8,7 @@ before_install:
 - '[ "${TRAVIS_PULL_REQUEST}" = "false" ] && openssl aes-256-cbc -K $encrypted_320e4a7e27b3_key
   -iv $encrypted_320e4a7e27b3_iv -in auth.js.enc -out test/resources/auth.js -d ||
   true'
+- npm install npm@latest -g
```

Comment:
Not sure about this line. Travis should already have `npm` installed and installing the "latest" every time had the potential to introduce breaking changes if they do a major release (unlikely, but still possible).

Did you need to add this to get it to work?